pkg/crc/machine/machine.go

```diff
@@ -41,25 +41,23 @@ func init() {
 	}
 }
 
-func fillClusterConfig(bundleInfo *bundle.CrcBundleInfo, clusterConfig *ClusterConfig) error {
+func getClusterConfig(bundleInfo *bundle.CrcBundleInfo) (*ClusterConfig, error) {
```

Comment:
I don't think this is related to adding crc contexts to ~/.kube/config?

Reply:
yes removed!